src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts

```diff
@@ -64,6 +65,10 @@ export class UbsAdminOrderDetailsFormComponent implements OnInit, OnChanges {
       this.isOrderNotTakenOut = curStatus?.key === OrderStatus.NOT_TAKEN_OUT;
     }
 
+    if (!prevStatus?.key && curStatus?.key === OrderStatus.BROUGHT_IT_HIMSELF) {
+      this.isDisabledWriteOffStation = true;
+    }
```

Comment:
Can you not use 'if' here?
this.isDisabledWriteOffStation = !prevStatus?.key && curStatus?.key === OrderStatus.BROUGHT_IT_HIMSELF

Reply:
yes, fixed it